app/src/main/java/com/infomaniak/mail/ui/main/thread/calendar/CalendarEventBannerView.kt

```diff
@@ -74,6 +79,12 @@ class CalendarEventBannerView @JvmOverloads constructor(
         }
 
         setAttendees(calendarEvent.attendees)
+
+        addToCalendarButton.setOnClickListener {
+            val navigationCallback = navigateToDownloadProgressDialog ?: return@setOnClickListener
+            val snackBarCallback = showSnackBar ?: return@setOnClickListener
+            attachment.openAttachment(context, navigationCallback, snackBarCallback)
```

Comment:
Doas opening the attachment is enoug to add it to the calendar ?

Reply:
Yeah, it is